README.md

```diff
@@ -26,7 +26,7 @@ Install the module with: `npm install gmsmith`
 var Gmsmith = require('gmsmith');
 
 // Create a new engine
-var gmsmith = new Gmsmith();
+var gmsmith = new Gmsmith({});
```

Comment:
Not passing an empty object in lead to an error in this line: https://github.com/twolfson/gmsmith/blob/master/lib/engine.js#L18

I could open an alternative PR to avoid that if you prefer.